### PR TITLE
[V3] test downloader

### PR DESCRIPTION
**Pull request by bobloy**
### Type

- [x] Bugfix
- [ ] Enhancement
- [ ] New feature

### Description of the changes

Option for fixing #1663

### CODE REVIEW

File: tests/cogs/downloader/test_downloader.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def test_existing_git_repo(tmpdir):
     assert clone_cmd[2] == "-b"
     assert clone_cmd[3] == "rewrite_cogs"
     assert clone_cmd[4] == repo_norun.url
-    assert "repos/squid" in clone_cmd[5]
+    assert "repos/squid" in clone_cmd[5] or "repos\\\\squid" in clone_cmd[5]
```
Comment (tekulvw) [inline tests/cogs/downloader/test_downloader.py]: This should compare against path parents.

For example:
```python
assert ["repos", "squid"] == pathlib.Path(clone_cmd[5]).parents[-2:]
```

**Review by tekulvw — DISMISSED** [review]

**Comment by bobloy** [comment]
Commited changes to address review. Used pathlib.Path().parts

**Comment by bobloy** [comment]
I have no idea what Travis is talking about with that build fail

**Comment by Tobotimus** [comment]
@bobloy We apologise for the confusion as Travis was installing a newer version of the stylechecking tool than the usual one. The issue has since been fixed, however we ask that you update your development requirements with `pipenv install --dev`.

Just as a side note, I would recommend running `pipenv install --dev` every time you pull/merge from V3/develop. Specifically, do it whenever you see that the `Pipfile.lock` file has been changed, for example here:

```bash
$ git pull
Updating 55afc7e..d3f406a
Fast-forward
 Pipfile.lock                           |  18 ++--
 ...
```

This usually means that a dependency has been pinned to a new version, and thus an update is necessary.

**Review by Tobotimus — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**